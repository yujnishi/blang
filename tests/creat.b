main() {
    auto NULL 0;

    creat("creat.0.tmp",0);
    if ( !fork() ) execl("/usr/bin/stat","stat","-c","%A %n","creat.0.tmp",NULL);
    wait();

    creat("creat.292.tmp",292); /* 0444 */
    if ( !fork() ) execl("/usr/bin/stat","stat","-c","%A %n","creat.292.tmp",NULL);
    wait();

    creat("creat.384.tmp",384); /* 0600 */
    if ( !fork() ) execl("/usr/bin/stat","stat","-c","%A %n","creat.384.tmp",NULL);
    wait();

}
