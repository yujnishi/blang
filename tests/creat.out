---------- creat.0.tmp
-r--r--r-- creat.292.tmp
-rw------- creat.384.tmp
